exercises/practice/phone-number/src/lib.cairo

```diff
@@ -1,3 +1,58 @@
 pub fn clean(phrase: ByteArray) -> ByteArray {
-    panic!("implement `clean`")
```

Comment:
Don't forget to revert _lib.cairo_ to a stub

Reply:
lib.cairo  currently shows as the stub locally and here. I think the site is showing you a previous iteration.

Reply:
I see now, it was probably as you described